cosmos/urls.py

```diff
@@ -14,6 +14,7 @@
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path("o/", include("oauth2_provider.urls", namespace="oauth2_provider")),
```

Comment:
maybe make this url a bit more descriptive? makes it easier to debug if later there are any issues with it